src/PieChart/PieArcSeries/PieArc.tsx

```diff
@@ -133,8 +134,10 @@ export const PieArc: FC<PieArcProps> = ({
     [gradient, id, color]
   );
 
+  const tooltipData = useMemo(() => ({ y: data.data.data, x: data.data.key }), [data]);
```

Comment:
Let's add a comment on why this is done

Reply:
This was just moved into a variable since the value had to be reused, it was there earlier for just the tooltip.